harness/tests/experiment/pytorch/test_examples.py

```diff
@@ -0,0 +1,213 @@
+# type: ignore
+import logging
+import pathlib
+import typing
+
+import pytest
+import torch
+from torch.distributed import launcher
+
+from tests.experiment import pytorch_utils, utils  # noqa: I100
+
+
+def test_pytorch_mnist(tmp_path: pathlib.Path) -> None:

```

Comment:
As a reader, I really like it when tests are named like <test>_<class_or_function_under_test>_<behavior_under_test>. It helps me understand what's being tested and what the desired outcomes of those tests are without having to read the code line by line. These stop with an analogue of <test>_<function>. Are they focused enough that it makes sense to also mention the tested behavior?